src/test/java/hudson/plugins/git/extensions/impl/CloneOptionHonorRefSpecTest.java

```diff
@@ -0,0 +1,136 @@
+package hudson.plugins.git.extensions.impl;
+
+import hudson.model.*;
+import hudson.plugins.git.*;
```

Comment:
General preference is to avoid wildcard imports for anything but the static import of assertions.  The contributing file says:

> New code should follow the [SCM API code style guidelines](https://github.com/jenkinsci/scm-api-plugin/blob/master/CONTRIBUTING.md#code-style-guidelines).